MicrosoftTeams.schema.json

```diff
@@ -805,6 +805,24 @@
       },
       "description": "When a group install scope is selected, this will define the default capability when the user installs the app",
       "additionalProperties": false
+    },
+    "configurableProperties": {
+      "type": "array",
+      "description": "A list of tenant configurable properties for an app",
+      "maxItems": 9,
```

Comment:
What is this max number used for?

Reply:
maxItems is the maximum number of items configurableProperties array can hold, which is all the enum values

Reply:
> What is this max number used for?

maxItems is the maximum number of items configurableProperties array can hold, which is all the enum values